runtime/common/src/registrar.rs

```diff
@@ -267,6 +291,10 @@ decl_module! {
 		///
 		/// Must be sent from a Signed origin that is able to have ParathreadDeposit reserved.
 		/// `code` and `initial_head_data` are used to initialize the parathread's state.
+		///
+		/// Unlike `register_para`, this function does check that the maximum code size
```

Comment:
Should we at least check the code size in the dispatchable `register_para` function?

Reply:
Yes, I guess it's meant to be called by governance, so if it fails because governance changes maximums in the meantime, it would be governance's fault.

Reply:
Fair enough.